mavros/src/plugins/hil.cpp

```diff
@@ -295,7 +295,7 @@ class HilPlugin : public plugin::PluginBase {
 		of.integrated_zgyro = int_gyro.z();
 		of.time_delta_distance_us = req->time_delta_distance_us;
 		of.distance = req->distance;
-		of.quality  = req->quality;
+		of.quality  = req->quality ;
```

Comment:
Modification in generated code looks like mistake. Cog will fail here :)
Use `cog.py -xr <file>` that usual `cog.py -cr <file>`